### ISSUE TITLE
Raycaster docs origin error

### ISSUE THREAD

**Issue by kirradchenko** [aframevr/aframe#5272]
**Description:**
While working with raycaster I needed to change the origin of raycaster line. In official documentation in the example, the origin is defined by position property, although the correct property is origin. In the Property section it is correctly defined property origin but in the example is not. Most people look at the example code and then copy it. I suggest correcting the example code.

- A-Frame Version: 1.4.0
- Platform / Device: Chrome
- Reproducible Code Snippet or URL: [raycaster - A-Frame](https://aframe.io/docs/1.4.0/components/raycaster.html)
`<a-entity raycaster="objects: .collidable" position="0 -0.9 0" rotation="90 0 0"></a-entity>`

**Comment by dmarcos** [comment]
Not sure I understand. position and origin are different things:

position is the position of the entity.
origin is the origin of the ray relative to the position of the entity as [described in the docs](https://aframe.io/docs/1.4.0/components/raycaster.html#properties_origin)

Is not that clear?

**Comment by kirradchenko** [comment]
Yes, sure I get it now. The position is the property of the entity component and origin of the raycaster. I am sorry.

I got confused because right now I am working with raycaster and I defined the position of the entity and it did not move the ray. Therefore I thought there is an issue with my definition of raycaster entity. Obviously, it worked with the origin property. 

Thanks for clarification. Have a nice day.
